src/components/ScreenWrapper/ScreenWrapperOfflineIndicators.tsx

```diff
@@ -47,12 +48,18 @@ function ScreenWrapperOfflineIndicators({
     addBottomSafeAreaPadding = true,
     addWideScreenBottomSafeAreaPadding = !!extraContent,
 }: ScreenWrapperOfflineIndicatorsProps) {
+    const [navigationBarType, setNavigationBarType] = useState<NavigationBarType>(CONST.NAVIGATION_BAR_TYPE.NONE);
```

Comment:
Similar suggestion here about `useState` and `useEffect`.

Reply:
done 🫡 - https://github.com/Expensify/App/pull/69535/commits/60e44612518aad9029a60df94c1546c0abf7a58e